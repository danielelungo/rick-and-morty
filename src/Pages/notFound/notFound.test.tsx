/* eslint-disable react/react-in-jsx-scope */

import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import NotFound from "./notFound";

describe("NotFound component", () => {
  test("renders NotFound Page", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <NotFound />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

NotFound;
