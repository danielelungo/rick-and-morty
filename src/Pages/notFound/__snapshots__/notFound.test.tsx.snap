// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders NotFound Page 1`] = `
<div>
  <h1>
    404: Page Non Found
  </h1>
  <a
    href="/"
    onClick={[Function]}
  >
    Go Back To Home Page
  </a>
</div>
`;
